### PR TITLE
Dv 640 send max

### PR DESCRIPTION
**Pull request by bajaj6390ankit**
This PR include: 
 - Add max button in input field
 - Add click functionality on that button and fill the input box with user balance
 - Update css for responciveness 
 - Test At local and generate PR

### CODE REVIEW

File: ui/src/components/NetworkDetails.js [hunk 1]
```diff
@@ -22,6 +22,25 @@ export const NetworkDetails = ({
   tokenName,
   getExplorerAddressUrl
 }) => {
+  console.log({
```
Comment (kourtellosg) [inline ui/src/components/NetworkDetails.js]: remove console log

File: ui/src/components/NetworkDetails.js [hunk 2]
```diff
@@ -35,6 +54,7 @@ export const NetworkDetails = ({
   const formattedBalance = isNaN(numeral(balance).format('0.00', Math.floor))
     ? numeral(0).format('0,0.00', Math.floor)
     : numeral(balance).format('0,0.000', Math.floor)
+    console.log(formattedBalance, totalAmount);
```
Comment (kourtellosg) [inline ui/src/components/NetworkDetails.js]: remove log

**Review by kourtellosg — CHANGES_REQUESTED** [review]
looks good - please remove logs and we can merge

File: ui/src/components/NetworkDetails.js (hunk 1 above)
Comment (bajaj6390ankit) [reply ui/src/components/NetworkDetails.js]: removed

File: ui/src/components/NetworkDetails.js (hunk 2 above)
Comment (bajaj6390ankit) [reply ui/src/components/NetworkDetails.js]: removed

**Review by kourtellosg — APPROVED** [review]

**Pull request merged**